### PR TITLE
[ADD] product_dependencies

### PR DESCRIPTION
**Pull request by max3903**
Transfered merge proposal from @joaoalf 

https://code.launchpad.net/~savoirfairelinux-openerp/openerp-product-attributes/product_dependencies/+merge/185082

### CODE REVIEW

File: product_dependencies/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+##############################################################################
+
+{
+    'name': 'Product Dependencies',
+    'version': '1.0',
+    'category': 'Product Management',
+    'description': """
+Allows products to have other products/categories as dependencies.
+
+This module is primarily used by the contract_isp_wizard module to create packages based on basic product selections but it can be used by others modules that manage product lists.""",
```
Comment (bwrsandman) [inline product_dependencies/__openerp__.py:30]: This line is clearly too long

File: product_dependencies/product.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp.osv import orm, fields
+
+
+class product_dependency(orm.Model):
```
Comment (bwrsandman) [inline product_dependencies/product.py:26]: This class/model should go into it's own file `product_dependency.py`

File: product_dependencies/product.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    _name = 'product.dependency'
+
```
Comment (bwrsandman) [inline product_dependencies/product.py:28]: Please add a `_description`

File: product_dependencies/product_dependencies_view.xml [hunk 4]
```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+	<record id="view_product_dependencies_product_form" model="ir.ui.view">
```
Comment (bwrsandman) [inline product_dependencies/product_dependencies_view.xml:4]: This record should be in a file named `product_view.xml` or `product_product_view.xml`

File: product_dependencies/product_dependencies_view.xml [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+				</tree>
+			    </field>
+			</group>
+		    </page>
+		</xpath>
+	    </field>
+	</record>
+	<record id="view_product_dependencies_form" model="ir.ui.view">
+	    <field name="name">product.dependencies.form</field>
+	    <field name="model">product.dependency</field>
```
Comment (bwrsandman) [inline product_dependencies/product_dependencies_view.xml:28]: This record should be in a file named `product_dependency.xml`

**Comment by pedrobaeza** [comment]
Any news on this PR?

**Comment by pedrobaeza** [comment]
As there's no answer, I close. Feel free to reopen in case there's interest.

**Pull request closed without merging**